pkg/shell/hosts_dialog.jsx

```diff
@@ -495,7 +501,10 @@
                        </Button>
                    </>}
             >
-                {body}
+                <Stack hasGutter>
+                    { this.props.dialogError && <ModalError dialogError={this.props.dialogError} />}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17221-20221202-001251-c9401003-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)